docs/DeveloperGuide.md

```diff
@@ -312,262 +312,282 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Use case ID: UC01**
 
-**Use case: Create fileInfo**
+**Use case: Tag a file**
 
 **MSS**
 
-1. User requests to tag a certain file using the absolute file path.
-2. HelloFile recognises the file path to be valid.
-3. HelloFile creates the tag and display the fileInfo.
+1. User requests to tag a file with a tag name, the file path and a label.
+2. HelloFile creates the tag and informs the user.
 
     Use case ends.
 
 **Extensions**
```

Comment:
Should we put that a label can be invalid as well in the extension?

Reply:
Ooo, ya, I will add that